[BUG] skill_resource_tests using incompatible version of neon_minerva

### Description

https://github.com/OpenVoiceOS/ovos-skill-easter-eggs/actions/runs/7536978652/job/20515205017?pr=35

```
FAILED action/github/test/test_skill_resources.py::TestSkillResources::test_skill_setup - AttributeError: 'EasterEggsSkill' object has no attribute '_core_lang'. Did you mean: 'core_lang'?
```

### Steps to Reproduce

```yaml
  skill_resource_tests:
    uses: neongeckocom/.github/.github/workflows/skill_test_resources.yml@master
```

### Relevant Code

_No response_

### Other Notes

_No response_

--- a/.github/workflows/skill_tests.yml
+++ b/.github/workflows/skill_tests.yml
@@ -13,8 +13,8 @@ jobs:
     with:
       neon_versions: "[3.8, 3.9, '3.10', '3.11']"
       ovos_versions: "[3.8, 3.9, '3.10', '3.11']"
-  # skill_intent_tests:
-  #   uses: neongeckocom/.github/.github/workflows/skill_test_intents.yml@master
+  skill_intent_tests:
+   uses: neongeckocom/.github/.github/workflows/skill_test_intents.yml@master
   skill_resource_tests:
     uses: neongeckocom/.github/.github/workflows/skill_test_resources.yml@master
   # skill_install_tests: # May be bugged from alpha

--- a/skill.json
+++ b/skill.json
@@ -1,6 +1,6 @@
 {
     "title": "",
-    "url": "https://github.com/OpenVoiceOS/ovos-skill-easter-eggs",
+    "url": "https://github.com/NeonDaniel/ovos-skill-easter-eggs",
     "summary": "",
     "short_description": null,
     "description": "Some funny things for OpenVoiceOS, mostly quotes from movies",
@@ -49,7 +49,7 @@
         "mikejgray (revival)"
     ],
     "skillname": "ovos-skill-easter-eggs",
-    "authorname": "OpenVoiceOS",
+    "authorname": "NeonDaniel",
     "foldername": null,
     "easter eggs": "pop culture references skill",
     "grandma mode": "This skill has some spicy language options. It starts in grandma mode by default. To disable grandma mode, say \"hurt me plenty.\" If you want to go back into grandma mode, say \"I'm too young to die.\""

--- a/test/test_intents.yaml
+++ b/test/test_intents.yaml
@@ -67,7 +67,7 @@ en-us:
     - stallions
     - wild stallions
     - wild stallion
-  malibu_stacy_intent:
+  malibu_stacey_intent:
     - malibu stacy
     - malley boo stay see
     - malla boo stay see

--- a/test/test_intents.yaml
+++ b/test/test_intents.yaml
@@ -57,7 +57,7 @@ en-us:
     - like glados
     - like portal
     - forrtal song
-  robotic_laws_intent:
+  robotics_laws_intent:
     - say the laws of robotics
     - do you know the rebotic laws
   rock_paper_scissors_lizard_spock_intent:

--- a/test/test_intents.yaml
+++ b/test/test_intents.yaml
@@ -1,35 +1,35 @@
 # Specify intents to test
 # Basic intent match tests only:
 en-us:
-  arnoldkeyword:
+  arnold_intent:
     - arnold say
     - terminator say
     - schwarzenegger say
-  stardatekeyword:
+  stardate_intent:
     - current stardate
     - current star date
     - current startdate
     - current start date
-  adultmodekeyword:
+  adult_mode_intent:
     - hurt me plenty
-  grandmamodekeyword:
+  grandma_mode_intent:
     - too young to die
-  benderkeyword:
+  bender_intent:
     - bender say
   conankeyword:
     - what is best in life
-  dukenukemkeyword:
+  duke_nukem_intent:
     - duke nukem say
     - duke newcom say
     - duke newum say
     - duke newham say
     - duke newcomb say
-  gladoskeyword:
+  glados_intent:
     - glados say
     - gladis say
     - gladdos say
     - gladys say
-  halkeyword:
+  hal_intent:
     - HAL say
     - HAL 9000 say
     - all say
@@ -38,16 +38,14 @@ en-us:
     - howl nine thousand say
     - how 9000 say
     - how nine thousand say
-  languagesyouspeakkeyword:
+  languages_you_speak_intent:
     - languages do you speak
     - languages can you speak
-  lawkeyword:
-    - law
-    - laws
-  podbaydoorskeyword:
+
+  pod_bay_doors_intent_intent:
     - open the pod bay doors
     - open pod bay doors
-  portalkeyword:
+  portal_intent:
     - sing portal
     - sing glados
     - glados sing
@@ -59,26 +57,17 @@ en-us:
     - like glados
     - like portal
     - forrtal song
-  roboticskeyword:
-    - robotic
-    - robotics
-  rockpaperscissorslizardspockkeyword:
+  robotic_laws_intent:
+    - say the laws of robotics
+    - do you know the rebotic laws
+  rock_paper_scissors_lizard_spock_intent:
     - rock paper scissors lizard spock
-  stardatekeyword:
-    - stardate
-    - star date
-    - startdate
-    - start date
-    - start day
-    - startday
-    - start eight
-    - star eat
-  billandtedkeyword:
+  bill_and_ted_intent:
     - stallion
     - stallions
     - wild stallions
     - wild stallion
-  malibustaceykeyword:
+  malibu_stacy_intent:
     - malibu stacy
     - malley boo stay see
     - malla boo stay see
@@ -88,38 +77,38 @@ en-us:
 de-de:
   adultmodekeyeword:
     - hurt me plenty
-  arnoldkeyword:
+  arnold_intent:
     - arnold sagen
     - der terminator sagen
     - schwarzenegger sagen
-  dukenukemkeyword:
+  duke_nukem_intent:
     - duke nukem sagen
     - duke newcom sagen
     - duke newum sagen
     - duke newham sagen
     - duke newcomb sagen
-  gladoskeyword:
+  glados_intent:
     - glados sagen
     - gladis sagen
-  grandmamodekeyword:
+  grandma_mode_intent:
     - too young to die
-  halkeyword:
+  hal_intent:
     - HAL sagen
     - HAL 9000 sagen
     - all sagen
     - all 9000 sagen
-  languagesyouspeakkeyword:
+  languages_you_speak_intent:
     - Welche Sprachen sprichst du
     - Welche Sprachen kannst du sprechen
-  lawkeyword:
-    - Rechte
-    - Gesetze
-    - kannst du Töten
-    - bist du böse
-  podbaydoorskeyword:
+#  lawkeyword:
+#    - Rechte
+#    - Gesetze
+#    - kannst du Töten
+#    - bist du böse
+  pod_bay_doors_intent_intent:
     - öffne das eingangstoor
     - öffne das Zugangstor
-  portalkeyword:
+  portal_intent:
     - Singe portal
     - singe glados
     - glados singt
@@ -130,12 +119,12 @@ de-de:
     - portal Lied
     - like glados
     - like portal
-  roboticskeyword:
-    - Roboter
-    - Robotic
-  rockpaperscissorslizardspockkeyword:
+#  roboticskeyword:
+#    - Roboter
+#    - Robotic
+  rock_paper_scissors_lizard_spock_intent:
     - Schere Stein Papier Echse Spock
-  stardatekeyword:
+  stardate_intent:
     - Sternenzeit
     - Stern Zeit
 

--- a/test/test_intents.yaml
+++ b/test/test_intents.yaml
@@ -16,7 +16,7 @@ en-us:
     - too young to die
   bender_intent:
     - bender say
-  conankeyword:
+  conan_intent:
     - what is best in life
   duke_nukem_intent:
     - duke nukem say
@@ -75,7 +75,7 @@ en-us:
     - what wood malley boo stay see do
     - what wood malla boo stay see do
 de-de:
-  adultmodekeyeword:
+  adult_mode_intent:
     - hurt me plenty
   arnold_intent:
     - arnold sagen

--- a/test/test_intents.yaml
+++ b/test/test_intents.yaml
@@ -59,7 +59,7 @@ en-us:
     - forrtal song
   robotics_laws_intent:
     - say the laws of robotics
-    - do you know the rebotic laws
+    - do you know the robotic laws
   rock_paper_scissors_lizard_spock_intent:
     - rock paper scissors lizard spock
   bill_and_ted_intent:

--- a/test/test_intents.yaml
+++ b/test/test_intents.yaml
@@ -42,7 +42,7 @@ en-us:
     - languages do you speak
     - languages can you speak
 
-  pod_bay_doors_intent_intent:
+  pod_bay_doors_intent:
     - open the pod bay doors
     - open pod bay doors
   portal_intent:
@@ -105,7 +105,7 @@ de-de:
 #    - Gesetze
 #    - kannst du Töten
 #    - bist du böse
-  pod_bay_doors_intent_intent:
+  pod_bay_doors_intent:
     - öffne das eingangstoor
     - öffne das Zugangstor
   portal_intent:

